### ISSUE TITLE
when processing processors, emit error if processor type is not found in the processor cfg

### ISSUE THREAD

**Issue by hellt** [karimra/gnmic#258]
consider the following config:

```yaml
outputs:
  prom:
    type: prometheus
    listen: :9273
    path: /metrics
    expiration: 60s # maximum lifetime of metrics in the local cache
    debug: false
    event-processors:
      - proc-convert-strings-to-int
      - set-interface-stats-metric-name

processors:
  set-interface-stats-metric-name:
    event-string:  # typo here
      value-names:
        - ".*"
      transforms:
        - trim-prefix:
            apply-on: "name"
            prefix: "/srl_nokia-interfaces:interface/statistics/"

```
It is hard to notice, that there is a typo in processor type (event-string**s**), because no error is emitted when output processes configured processors

**Comment by karimra** [comment]
will be addressed in #252

**Comment by hellt** [comment]
yes, it know hard stops if any processor is not found, which is good

```
gnmic 2021/01/07 13:57:08.872486 using config file /root/container-lab/lab-examples/telemetry01/gnmic-config.yml
config 2021/01/07 13:57:08.873269 validating processor "proc-convert-strings-to-int" config
Error: unknown processors type: event-converts
```
